test/integration/tests/abrmd_policycountertimer.sh

```diff
@@ -0,0 +1,88 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+source helpers.sh
+
+cleanup() {
+
+  rm -f session.ctx prim.ctx key.pub key.priv key.ctx policy.countertimer.minute
+
+  if [ "$1" != "no-shut-down" ]; then
+     shut_down
+  fi
+}
+
+trap cleanup EXIT
+
+start_up
+
+cleanup "no-shut-down"
+
+tpm2_clear
+
+#
+# Create a sealing object with a policy that evaluates for first minute after
+# TPM restart. NOTE the time is 60000 milliseconds.
+#
+SIXTYTHOUSANDMILLISECONDS=60000
+
+tpm2_startauthsession -S session.ctx
+tpm2_policycountertimer -S session.ctx -L policy.countertimer.minute --ult \
+"$SIXTYTHOUSANDMILLISECONDS"
+
+tpm2_flushcontext session.ctx
+
+tpm2_createprimary -C o -c prim.ctx -Q
+
+echo "SUPERSECRET" | \
+tpm2_create -Q -u key.pub -r key.priv -i- -C prim.ctx \
+-L policy.countertimer.minute -a "fixedtpm|fixedparent" -c key.ctx
+
+#
+# ASSUMING 1 minute hasn't elapsed since clear, Try unseal in the first minute
+# -- Should pass
+#
+tpm2_startauthsession -S session.ctx --policy-session
+
+tpm2_policycountertimer -S session.ctx -L policy.countertimer.minute --ult \
+"$SIXTYTHOUSANDMILLISECONDS"
+
+tpm2_unseal -c key.ctx -p session:session.ctx
+
+tpm2_flushcontext session.ctx
+
+#
+# Test if a policycountertimer evaluates with the clock
+#
+tpm2_clear
+tpm2_startauthsession -S session.ctx --policy-session
+tpm2_policycountertimer -S session.ctx --ult clock $SIXTYTHOUSANDMILLISECONDS
```

Comment:
The code looks like it can process multiple things, like clock 60000 restart 2, etc, is this true? I don't see you testing that. Also, I would make these key=value arguments like dd so it's more clear.

```C
for (i=0; i < argc; i++) {
  char *a = argv[i]
  char *split = strchr(a, '=');
  if (!split) {
    // error
  }
  split[0] = '\0';
  const char *value = split + 1;
  const char *key = a;

  if (!value[0]) {
    //error
  }

  // look up key and process value
}
```

Reply:
Policycountertimer is an immediate assertion that will fail instantly in a non-trial policy session.

Reply:
Huh? But can't multiple fields be set?

Reply:
No, they can't be. The last one parsed will replace operand buffer content. This is why I error out if more than two arguments are received. Although, I see your point - I will resort to key=value mechanism and obviate the do{}while.

Reply:
Ahh I missed that.